### ISSUE TITLE
EDB open function - needs code for checking if open was OK?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jwhiddon/arduino-edb#3]
```
I am looking in the edb lib... very interesting...!

With edb we can open a edb table from the EEPROM... How can we make sure that 
the opened edb object holds the correct data (for the corresponding record 
structure)?

I mean how can I confirm that the already opened data structure (EEPROM) and 
the EDB object (opened) use the absolute same record structure?

I even try to open a structure that was not existed (in EEPROM) and the 
function returned "EDB_OK"

I think that something is missing here...
```

Original issue reported on code.google.com by `ago...@gmail.com` on 26 Feb 2011 at 5:43

**Comment by joscha** [comment]
I think someone made a fix for this problem here: https://github.com/firebull/arduino-edb/commit/46e8b85f4894962cf165dea1dc8b3460cb9f907b - @jwhiddon should I try and port this back?

**Comment by joscha** [comment]
fixed in 1954c208c0f2652c21041a0a2d9b43c22db2cfc1 and e981a60
